src/confab/agent/runner_test.go

```diff
@@ -99,7 +98,7 @@ var _ = Describe("Runner", func() {
 					},
 					{
 						Action: "agent-runner.cleanup.remove.failed",
-						Error:  expectedError,
+						Error:  errors.New(err.Error()),
```

Comment:
Because in the [implementation](https://github.com/greenhouse-org/consul-release/blob/c7aee13e970e94d7d99d0abaede904ecf19985be/src/confab/agent/runner.go#L174) the error is marshaled into a `errors.New()`. 

We had to change the code to return the real `os.PathError` rather than the marshaled form so that we could use `os.IsNotExist` to assert on it in a cross-platform way, rather than relying on os-specific string comparisons.